sdks/python/setup.py

```diff
@@ -59,18 +80,18 @@ def get_version():
     'oauth2client>=2.0.1',
     'protorpc>=0.9.1',
     'python-gflags>=2.0',
-    'pyyaml>=3.10',
-    ]
-
+    'pyyaml>=3.10'
```

Comment:
i think we usually prefer to leave the `,` at the end of the list, so that lines can be added and removed easily without stressing over whether they're last in the file. Deliberate removal?